test/unit/auth/user-import-builder.spec.ts

```diff
@@ -861,9 +864,21 @@ describe('UserImportBuilder', () => {
       };
       const userImportBuilder = new UserImportBuilder(
         testUsers, validOptions as any, userRequestValidatorWithMultipleErrors);
-      expect(userImportBuilder.buildResponse(failingServerResponse))
-        .to.deep.equal(mixedErrorUserImportResponse);
+      expectUserImportResult(
+        userImportBuilder.buildResponse(failingServerResponse),
+        mixedErrorUserImportResponse);
     });
+
+    function expectUserImportResult(result: UserImportResult, expected: UserImportResult): void {
+      expect(result.successCount).to.equal(expected.successCount);
+      expect(result.failureCount).to.equal(expected.failureCount);
+      expect(result.errors.length).to.equal(expected.errors.length);
+      result.errors.forEach((err, idx) => {
+        const want = expected.errors[idx];
+        expect(err.index).to.equal(want.index);
+        expect(err.error).to.deep.include(want.error);
+      });
+    }
```

Comment:
Consider extracting this to a common utility, since it's used in multiple files

Reply:
Done. Exported from this module, and imported into the other.